src/Ghcid.hs

```diff
@@ -308,7 +308,10 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
         putStrLn $ "\nNo files loaded, GHCi is not working properly.\nCommand: " ++ command
         exitFailure
 
-    restart <- return $ nubOrd $ restart ++ [x | LoadConfig x <- messages]
+    let stripTrailingSep = reverse . dropWhile (`elem` pathSeparators) . reverse
```

Comment:
This is just https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Posix.html#v:dropTrailingPathSeparator ?